### PR TITLE
Add draft of Full Change Log for 0.8.0 blog post

### PR DESCRIPTION
**Pull request by colepoirier**
Hey @cart, I tried my best to organize the sections by priority and add markdown strikethrough formatting and put at the bottom of each section the things that I thought should not be included. Obviously you'll have differing opinions on some of the determinations and changes I made; I hope this is still helpful. I didn't format the links yet because that will be easier to do once all of the inclusion determinations have been made by you.

### CODE REVIEW

File: content/news/2022-07-29-bevy-0.8/index.md [hunk 1]
```diff
@@ -1208,3 +1208,408 @@ A huge thanks to the **X contributors** that made this release (and associated d
...
+- Add benches for simple run criteria (#4196)
+- Add a random access get_component benchmark (#4607)
+- Add `bevy_reflect::{List, Map, Struct}` benchmarks (#3690)
+- Add benchmarks for schedule dependency resolution (#4961)
+- Use const generics to add wider variants of our existing ECS benchmarks (#5123)
+- Add `TextureAtlas` stress test based on `many_sprites` and `sprite_sheet` examples (#5087)
+- Add an `assert_is_exclusive_system` function (#5275)
+- Add keyboard key to mouse control for `scene_viewer` example (#4411)
+- Introduce tests for `derive(WorldQuery)` (#4625)
+- Add an example to test small window sizes (#3597)
```
Comment (alice-i-cecile) [inline content/news/2022-07-29-bevy-0.8/index.md]: This is internal.

File: content/news/2022-07-29-bevy-0.8/index.md [hunk 2]
```diff
@@ -1208,3 +1208,408 @@ A huge thanks to the **X contributors** that made this release (and associated d
...
+- Add more `SAFETY` comments and lint for missing ones in `bevy_ecs` (#4835)
+- Add benches for simple run criteria (#4196)
+- Add a random access get_component benchmark (#4607)
+- Add `bevy_reflect::{List, Map, Struct}` benchmarks (#3690)
+- Add benchmarks for schedule dependency resolution (#4961)
+- Use const generics to add wider variants of our existing ECS benchmarks (#5123)
+- Add `TextureAtlas` stress test based on `many_sprites` and `sprite_sheet` examples (#5087)
+- Add an `assert_is_exclusive_system` function (#5275)
+- Add keyboard key to mouse control for `scene_viewer` example (#4411)
+- Introduce tests for `derive(WorldQuery)` (#4625)
```
Comment (alice-i-cecile) [inline content/news/2022-07-29-bevy-0.8/index.md]: Internal.

File: content/news/2022-07-29-bevy-0.8/index.md [hunk 3]
```diff
@@ -1208,3 +1208,408 @@ A huge thanks to the **X contributors** that made this release (and associated d
...
+- bevy_reflect: add statically available type info for reflected types (#4042)
+- Add more `SAFETY` comments and lint for missing ones in `bevy_ecs` (#4835)
+- Add benches for simple run criteria (#4196)
+- Add a random access get_component benchmark (#4607)
+- Add `bevy_reflect::{List, Map, Struct}` benchmarks (#3690)
+- Add benchmarks for schedule dependency resolution (#4961)
+- Use const generics to add wider variants of our existing ECS benchmarks (#5123)
+- Add `TextureAtlas` stress test based on `many_sprites` and `sprite_sheet` examples (#5087)
+- Add an `assert_is_exclusive_system` function (#5275)
+- Add keyboard key to mouse control for `scene_viewer` example (#4411)
```
Comment (alice-i-cecile) [inline content/news/2022-07-29-bevy-0.8/index.md]: Internal.

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Benchmarks, examples and tests are all internal and should be removed.

**Comment by colepoirier** [comment]
> Benchmarks, examples and tests are all internal and should be removed.

I think I got them all.

File: content/news/2022-07-29-bevy-0.8/index.md [hunk 4]
```diff
@@ -1208,3 +1208,408 @@ A huge thanks to the **X contributors** that made this release (and associated d
...
+- Remove `Serialize` impl for `dyn Array` and friends (#4780)
+- Re-enable `#[derive(TypeUuid)]` for generics (#4118)
+- Move primitive type registration into `bevy_reflect` (#4844)
+- Implement reflection for more `glam` types (#5194)
+- Make `reflect_partial_eq` return more accurate results (#5210)
+- Make public macros more robust with `$crate` (#4655)
+- Ensure that the parent is always the expected entity (#4717)
+- Support returning data out of `with_children` (#4708)
+- Remove `EntityMut::get_unchecked` (#4547)
+
```
Comment (alice-i-cecile) [inline content/news/2022-07-29-bevy-0.8/index.md]:
```suggestion
```

File: content/news/2022-07-29-bevy-0.8/index.md [hunk 5]
```diff
@@ -1208,3 +1208,408 @@ A huge thanks to the **X contributors** that made this release (and associated d
...
+~~- Remove double blank line from component docs (#5102)~~
+~~- Default image sampler doc fix (#5047)~~
+~~- Remove spirv-reflect from readme (#5192)~~
+~~- remove an unused import in release (#5320)~~
+~~- remove component and resource suffixes from reflect structs (#5219)~~
+~~- Allow unicode license (#5337)~~
+~~- Remove some unused dependencies (#4544)~~
+~~- Misc dependency improvements (#4545)~~
+~~- Decouple some dependencies (#3886)~~
+~~- Update ndk-glue requirement from 0.5 to 0.6 (#3624)~~
```
Comment (alice-i-cecile) [inline content/news/2022-07-29-bevy-0.8/index.md]: IMO dependency bumps should be listed, but likely in a separate section (or all together) to reduce noise.

Comment (colepoirier) [reply content/news/2022-07-29-bevy-0.8/index.md]: Sounds good to me, but I'll leave that for @cart.

**Review by cart — APPROVED** [review]
This was extremely helpful. Much appreciated @colepoirier!

**Comment by colepoirier** [comment]
Awesome! Glad it was helpful :)

**Pull request merged**